### DISCUSSION TITLE
Pinia state property getter not reactive to changes?

### DISCUSSION THREAD

**Discussion by Brunnn** [vuejs/pinia#2306] [Help and Questions]
I'm using the `this.$patch()` to perform a change in the state of the store, but it seems that this is not updating the store's properties direct getters, see below the code insided the vue component:
![image](https://github.com/vuejs/pinia/assets/42523274/eff89181-4388-4a0e-bfcf-9ea57f46e350)
Here i call an action that will update the state, see the action code:
![image](https://github.com/vuejs/pinia/assets/42523274/27ee0dbb-c512-46eb-837f-e9a84db70cd0)

Those are the logs in the devtools:
![image](https://github.com/vuejs/pinia/assets/42523274/40690955-cdd8-4032-9608-64180486f7cf)

Is that the expected behavior?

**Comment by posva** [comment]
Put code, not screenshots and show all of the relevant code of the store: state/getters. Note you cannot have a state and a getter with the same name BTW
